server/routes/ad.ts

```diff
@@ -1004,6 +1012,9 @@ export default class AdService {
               featureData.data != null && featureData.data !== 'NaN'
                 ? toFixedNumberForAnomaly(Number.parseFloat(featureData.data))
                 : 0,
+            name: featureData.feature_name, 
+            expectedValue: result.anomaly_grade > 0 
+            ? this.getExpectedValue(result._source.expected_values[0].value_list, featureData.feature_id) : featureData.data
```

Comment:
if `result._source.expected_values` is empty this will throw an NPE

Reply:
it's replying on the `result.anomaly_grade > 0` check. but I can certainly add more checks to make sure the page won't crash